### DISCUSSION TITLE
tnsnames.ora support?

### DISCUSSION THREAD

**Discussion by algosoup** [sijms/go-ora#459] [Q&A]
Is there a way of using the tnsnames.ora aliases (sqlnet) to create connections?

**Comment by sijms** [comment]
parts of sqlnet.ora that driver supported are translated as url parameters

**Reply by algosoup** [reply]
Cool. Do you have an example? I always get port missing errors

**Comment by sijms** [comment]
```golang
option = map[string]string {
    "trace file": "trace.log",
    // other options can be found in README.md
}
connUrl := go_ora.BuildUrl(server, port, service, user, password, options)
db, err := sql.Open("oracle", connUrl)
```

**Reply by bck01215** [reply]
How is this used? The port, server and service should be aliased from the .ora file?
